script/restyle-images.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I feel quite strongly about not having this here.
we can regenerate thumbnails, but not change the styles this way.